p2p-specs/p2p-interface.md

```diff
@@ -165,26 +167,40 @@ For any optional queueing, Bundlers SHOULD maintain maximum queue sizes to avoid
 
 #### Global topics
 
-There are two primary global topics used to propagate user operations (`UserOperationWithEntryPoint`)
-and aggregate user operation hashes (`NewPooledUserOperationsHashes`) to all nodes on the network.
+The primary global topics used to propagate user operations to all nodes on the network is `UserOperationWithEntryPoint`.
 
 ##### `user_ops_with_entry_point`
 
 The `user_ops_with_entry_point` topic is the concatenation of EntryPoint address and UserOperation message serialized using SSZ
```

Comment:
a `user_ops_with_entry_point` is entryPoint and a **LIST OF** serialized UserOperation objects
we don't have a specific serialization of a single UserOp (its an entrypoint with a list of a single UserOp)

Reply:
Also: I'm confused, as we say here we use SSZ, and at the top it is mentioned we use `protobuf`.

Reply:
> a `user_ops_with_entry_point` is entryPoint and a **LIST OF** serialized UserOperation objects we don't have a specific serialization of a single UserOp (its an entrypoint with a list of a single UserOp)

Noted and fixed.

Reply:
> Also: I'm confused, as we say here we use SSZ, and at the top it is mentioned we use `protobuf`.

We are re-using the same methodology used by Ethereum consensus clients, where both SSZ and protobuf work in tandem.